plugins/org.eclipse.xtext/src/org/eclipse/xtext/generator/JavaIoFileSystemAccess.java

```diff
@@ -235,4 +235,10 @@ public CharSequence readTextFile(String fileName, String outputCfgName) throws R
 			throw new RuntimeIOException(e);
 		}
 	}
+
+	@Override
+	public boolean exists(String fileName, String outputConfigurationName) throws RuntimeIOException {
+		File file = getFile(fileName, outputConfigurationName);
+		return file!=null && file.exists();
```

Comment:
Yes, that should be correct and semantically equivalent to IFile.exists. Could you add a test for that case, too and add a remark to the JavaDoc?